packages/office-ui-fabric-react/src/components/TextField/examples/TextField.Basic.Example.tsx

```diff
@@ -2,17 +2,16 @@ import * as React from 'react';
 import { TextField, MaskedTextField } from 'office-ui-fabric-react/lib/TextField';
 import { Stack, IStackProps, IStackStyles } from 'office-ui-fabric-react/lib/Stack';
 
+const stackTokens = { childrenGap: 50 };
 const stackStyles: Partial<IStackStyles> = { root: { width: 650 } };
+const columnProps: Partial<IStackProps> = {
+  tokens: { childrenGap: 15 },
+  styles: { root: { width: 300 } },
+};
 
 export const TextFieldBasicExample: React.FunctionComponent = () => {
-  // TextFields don't have to be inside Stacks, we're just using Stacks for layout
```

Comment:
Could you re-add this comment that was lost in translation?

Reply:
My 2c; I'd think it's probably ok to leave it out and keep the example simple, seems obvious they're there for layout. We use Stack elsewhere without the clarification.

Reply:
Sounds good to me, I'll resolve this comment.